### PR TITLE
Moved Fact to correct path and using YUM rather than subscription-manager

### PR DESCRIPTION
**Pull request by tux-o-matic**
Solution for #18 and moved the Ruby file to the expected path to be executed by Facter.

### CODE REVIEW

**Comment by bastelfreak** [comment]
as discussed in the issue, could you please migrate it to awk and something with only one pipe?

File: lib/facter/rhsm_repos.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Facter.add('rhsm_repos') do
   setcode do
     repos = Array.[]
-    repo_list = Facter::Core::Execution.exec('subscription-manager repos --list-enabled | grep "Repo ID" | cut -d " " -f 5')
+    repo_list = Facter::Core::Execution.exec('yum repolist | awk '/Red Hat/ {print $1}'')
```
Comment (bastelfreak) [inline lib/facter/rhsm_repos.rb]: single quotes in single quotes doesn't work. Can you excahnge the outta quotes with double quotes? That should work.

**Comment by tux-o-matic** [comment]
@bastelfreak My bad, should be good to go now.

**Comment by bastelfreak** [comment]
thanks \o/

**Pull request merged**